### PR TITLE
Enable filtering on position

### PR DESCRIPTION
**Pull request by martinthomson**
The legend and positions are a button, but they didn't do anything.  This makes them do the obvious thing.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -224,16 +224,18 @@ <h3>legend</h3>
...
                   if (type !== "display") {
                     return data;
                   }
                   if (data !== "" && ! status_styles.hasOwnProperty(data)) {
-                    console.log("WARNING: unrecognised position '" + data + "' on " + row.name)
+                    console.log("WARNING: unrecognised position '" + data + "' on " + row.name);
                   }
-                  return "<button type='button' class='btn btn-" + status_styles[data] + " btn-xs'>" + data + "</button>"
+                  return "<button type='button' class='btn btn-xs position btn-" + status_styles[data] +
+                    "' onclick='filterPosition'>" + data + "</button>";
```
Comment (zcorpan) [inline index.html:238]:
```suggestion
                    "' onclick='filterPosition(event)'>" + data + "</button>";
```

Comment (martinthomson) [reply index.html:238]: That's my mistake; I should have reverted this one.  I'm using a different (better?) technique to attach the event handler below.

```suggestion
                    "'>" + data + "</button>";
```

**Comment by zcorpan** [comment]
This works but there's no way to get back to showing all rows, short of reloading the page. Maybe there should be a "clear filters" button to reset this and clear the search field?

**Comment by martinthomson** [comment]
I added a button.  PTAL.

**Review by zcorpan — APPROVED** [review]

**Pull request merged**